torch/csrc/monitor/python_init.cpp

```diff
@@ -85,31 +85,128 @@ void initMonitorBindings(PyObject* module) {
 
   auto m = rootModule.def_submodule("_monitor");
 
-  py::enum_<Aggregation>(m, "Aggregation")
-      .value("VALUE", Aggregation::NONE)
-      .value("MEAN", Aggregation::MEAN)
-      .value("COUNT", Aggregation::COUNT)
-      .value("SUM", Aggregation::SUM)
-      .value("MAX", Aggregation::MAX)
-      .value("MIN", Aggregation::MIN)
+  py::enum_<Aggregation>(
+      m,
+      "Aggregation",
+      R"DOC(
+        These are the types of aggregations that can be used as part of the
+        stats.
+      )DOC")
+      .value(
+          "VALUE",
+          Aggregation::NONE,
+          R"DOC(
+            VALUE returns the last value to be added.
+          )DOC")
+      .value(
+          "MEAN",
+          Aggregation::MEAN,
+          R"DOC(
+            MEAN computes the mean of all the added values.
+          )DOC")
+      .value(
+          "COUNT",
+          Aggregation::COUNT,
+          R"DOC(
+            COUNT returns the number of added values.
+          )DOC")
+      .value(
+          "SUM",
+          Aggregation::SUM,
+          R"DOC(
+            SUM returns the sum of the added values.
+          )DOC")
+      .value(
+          "MAX",
+          Aggregation::MAX,
+          R"DOC(
+            MAX returns the max of the added values.
+          )DOC")
+      .value(
+          "MIN",
+          Aggregation::MIN,
+          R"DOC(
+            MIN returns the min of the added values.
+          )DOC")
       .export_values();
 
   py::class_<Stat<double>>(m, "Stat")
-      .def("add", &Stat<double>::add)
-      .def("get", &Stat<double>::get)
-      .def_property_readonly("name", &Stat<double>::name)
-      .def_property_readonly("count", &Stat<double>::count);
+      .def(
+          "add",
+          &Stat<double>::add,
+          py::arg("v"),
+          R"DOC(
+            Adds a value to the stat to be aggregated according to the
+            configured stat type and aggregations.
+          )DOC")
+      .def(
+          "get",
+          &Stat<double>::get,
+          R"DOC(
+            Returns the current value of the stat primary for testing purposes.
+            If the stat has logged this will be zero.
+          )DOC")
+      .def_property_readonly(
+          "name",
+          &Stat<double>::name,
+          R"DOC(
+            The name of the stat that was set during creation.
+          )DOC")
+      .def_property_readonly(
+          "count",
+          &Stat<double>::count,
+          R"DOC(
+            Number of data points that have currently been collected. Resets
+            once the event has been logged out.
+          )DOC");
 
-  py::class_<IntervalStat<double>, Stat<double>>(m, "IntervalStat")
-      .def(py::init<
-           std::string,
-           std::vector<Aggregation>,
-           std::chrono::milliseconds>());
+  py::class_<IntervalStat<double>, Stat<double>>(
+      m,
+      "IntervalStat",
+      R"DOC(
+        IntervalStat is a Stat that logs the stat once every `window_size`
```

Comment:
is a Stat that logs ~the stat~ once every

also mention that this is in milliseconds?

Reply:
updated to show the function signature since it takes a timedelta